src/ray/raylet/scheduling_queue.cc

```diff
@@ -69,7 +69,13 @@ bool TaskQueue::AppendTask(const TaskID &task_id, const Task &task) {
   task_map_[task_id] = list_iterator;
   // Resource bookkeeping
   total_resource_load_.AddResources(task.GetTaskSpecification().GetRequiredResources());
-  resource_load_by_shape_[task.GetTaskSpecification().GetSchedulingClass()]++;
+  const auto &scheduling_class = task.GetTaskSpecification().GetSchedulingClass();
+  resource_load_by_shape_[scheduling_class]++;
+
+  int64_t backlog_size = task.BacklogSize();
+  if (backlog_size >= 0) { // Poor man's version of RayConfig::instance().report_worker_backlog()
```

Comment:
>0?

Reply:
-1 in the stand-in value if we aren't reporting the backlog, but i guess we don't actually need to track anything if the backlog is exactly 0 either.

Reply:
Oh yeah. That's what I meant. It was supposed to be `>0`, but it was auto-formatted by github lol